### DISCUSSION TITLE
node:buffer

### DISCUSSION THREAD

**Discussion by babita-web** [vercel/next.js#33982] [App Router]
node:buffer
Module build failed: UnhandledSchemeError: Reading from "node:buffer" is not handled by plugins (Unhandled scheme).
Webpack supports "data:" and "file:" URIs by default.
You may need an additional plugin to handle "node:" URIs.

**Comment by alex-semenyuk** [comment]
Similar to this issue https://github.com/sindresorhus/file-type/issues/502

**Comment by 0xmalchik** [comment]
Same issue here working with node-fetch lib

**Comment by marco910** [comment]
Same issue here. Also using `node-fetch`

**Comment by succulentpup** [comment]
Same issue here, working with node-fetch.

**Comment by cujarrett** [comment]
Same issue for me with `node-fetch` `3.2.3` and `react` `18.0.0`.

**Comment by rajashekar63** [comment]
same issue for me with `node-fetch 3.2.3` and `react 17.0.2` with `react-scripts 5.0.1`

**Comment by patchmix** [comment]
same issue for me with `node-fetch@3.2.3` and `next@12.1.4`

**Comment by gordun209-hub** [comment]
same with node-fetch@3.2.3 and next 12.1.5

**Comment by sndrws** [comment]
having this issue with node-fetch

**Comment by palpablejc** [comment]
I'm having the same issue with node-fetch@3.2.4 and angular 12.1.5

**Comment by paullewisn** [comment]
As a workaround to the above, downgrading `node-fetch@2.x.x` clears the problem

**Reply by SamuelGHill** [reply]
I downgraded to `"node-fetch": "^2.6.1"` and now have no issues.

**Reply by phaethon5882** [reply]
thx brother, you made my day!

**Reply by NikkiMather** [reply]
Thanks, worked for me :)

**Reply by hamed8993** [reply]
> As a workaround to the above, downgrading `node-fetch@2.x.x` clears the problem

so thanks bro.
for other ones that may have problem by NEXT.JS@13.1.2 :

i has same problem with NEX.JS@13.1.2 and fetch-node@3.3.0
by installing fetch-node@2.6.1 , problem solved.

**Comment by jtebaan** [comment]
Here solution, works for me with next.js 13 https://github.com/vercel/next.js/issues/32935

**Comment by ShubhamMatoliya** [comment]
I am facing same error with node: util.  @nextJS 13

**Reply by ShubhamMatoliya** [reply]
Module build failed: UnhandledSchemeError: Reading from "node:util" is not handled by plugins (Unhandled scheme).
Webpack supports "data:" and "file:" URIs by default.
You may need an additional plugin to handle "node:" URIs.
Import trace for requested module:
node:util

**Comment by jorchRomera** [comment]
I have the same issue using Apollo Client with the app router.

**Comment by TommeZ** [comment]
I'm having the same issue when using @gradio/client

**Reply by Bulalu** [reply]
this seems to fix it 
https://stackoverflow.com/a/76522223

**Reply by TommeZ** [reply]
Gradio team are attempting to resolve this here - https://github.com/gradio-app/gradio/issues/5583

**Comment by netrolite** [comment]
Same issue with file-type ^18.5.0
